### PR TITLE
Improvement / Add safe SDK to setMetadataParams script

### PR DESCRIPTION
**Pull request by BravoNatalie**
## Describe Changes

I made this more better by doing ...

- Add safe wallet support to `scripts/countdown_erc721/5-setMetadataParams.ts`
- Remove price oracle call from purchase script

image
image

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All Hardhat tests are passing

### CODE REVIEW

**Review by Yashiru — APPROVED** [review]

**Review by alexanderattar — APPROVED** [review]
This looks great

File: scripts/countdown_erc721/README.md [hunk 1]
```diff
@@ -120,3 +120,27 @@ This is a straightforward script that invokes the contract `purchase` function u
 ```sh
 ts-node ./scripts/countdown_erc721/4-purchase.ts 
 ```
+
+## Script 5: Set Metadata 
+
+This is a straightforward script that invokes the contract `setMetadataParams` function using hardcoded values as parameters. This script also allows the execution to be made using a Safe wallet.
+
+PS: Please ensure that you execute the script using a private key belonging to one of the Safe address owners if you're running the script with the Safe wallet.
```
Comment (sogoiii) [inline scripts/countdown_erc721/README.md:129]: @BravoNatalie i like this call out. Is there a way to validate this before its submitted? 

What happens if i used an unapproved wallet? What kind of error do i get?

File: scripts/countdown_erc721/README.md [hunk 2]
```diff
@@ -120,3 +120,27 @@ This is a straightforward script that invokes the contract `purchase` function u
...
+1. **Set the static values:**
+   - Configure the static values:
+     - `contractAddress`: The CountdownERC721 address
+     - `params`: The metadata params that are going to be updated.
+     - `safeAddress`: (OPTIONAL) This is the Safe Wallet address, and should only be filled if the Safe wallet is going to be used.
+  
+2. **Run the script update the owner:**
+   - To execute the script using a Safe Wallet pass the '--safe' param:
+     ```sh
+     ts-node ./scripts/countdown_erc721/4-purchase.ts --safe
```
Comment (sogoiii) [inline scripts/countdown_erc721/README.md]: @BravoNatalie this has the `purchase.ts` in the command line. It should be the setMetadataParameters file right?

File: scripts/countdown_erc721/README.md [hunk 3]
```diff
@@ -120,3 +120,27 @@ This is a straightforward script that invokes the contract `purchase` function u
...
+     ```
+   - To execute the script with a standard wallet just run:
+     ```sh
+     ts-node ./scripts/countdown_erc721/4-purchase.ts
```
Comment (sogoiii) [inline scripts/countdown_erc721/README.md]: @BravoNatalie same as above, it should not be `purchase.ts` right?

**Review by sogoiii — CHANGES_REQUESTED** [review]

File: scripts/countdown_erc721/README.md [hunk 4]
```diff
@@ -120,3 +120,27 @@ This is a straightforward script that invokes the contract `purchase` function u
 ```sh
 ts-node ./scripts/countdown_erc721/4-purchase.ts 
 ```
+
+## Script 5: Set Metadata
```
Comment (alexanderattar) [inline scripts/countdown_erc721/README.md:125]: @BravoNatalie can you add this latest script to the top of the file where you have the scripts listed?

File: scripts/countdown_erc721/README.md (hunk 1 above)
Comment (BravoNatalie) [reply scripts/countdown_erc721/README.md:129]: The SDK does not allow it.

image

**Pull request merged**